tdigest.go

```diff
@@ -335,3 +345,96 @@ func (cs *centroidSet) MergeInto(other TDigest) {
 		other.Add(c.mean, c.count)
 	}
 }
+
+func (cs *centroidSet) MarshalBinary() ([]byte, error) {
+	buf := bytes.NewBuffer(nil)
+	w := &stickyErrWriter{w: buf, err: nil}
+	binary.Write(w, binary.LittleEndian, int32(1))
```

Comment:
Please label this magic number. It could be a line within this function of `const encodingVersion int32 = 1`.

Reply:
Yep, done